eng/Baseline.Designer.props

```diff
@@ -2,28 +2,28 @@
 <Project>
   <PropertyGroup>
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
-    <AspNetCoreBaselineVersion>5.0.3</AspNetCoreBaselineVersion>
+    <AspNetCoreBaselineVersion>5.0.4</AspNetCoreBaselineVersion>
   </PropertyGroup>
   <!-- Package: AspNetCoreRuntime.5.0.x64-->
   <PropertyGroup Condition=" '$(PackageId)' == 'AspNetCoreRuntime.5.0.x64' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <!-- Package: AspNetCoreRuntime.5.0.x86-->
   <PropertyGroup Condition=" '$(PackageId)' == 'AspNetCoreRuntime.5.0.x86' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <!-- Package: dotnet-sql-cache-->
   <PropertyGroup Condition=" '$(PackageId)' == 'dotnet-sql-cache' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <!-- Package: Microsoft.AspNetCore.ApiAuthorization.IdentityServer-->
   <PropertyGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' ">
-    <BaselinePackageVersion>5.0.3</BaselinePackageVersion>
+    <BaselinePackageVersion>5.0.4</BaselinePackageVersion>
   </PropertyGroup>
   <ItemGroup Condition=" '$(PackageId)' == 'Microsoft.AspNetCore.ApiAuthorization.IdentityServer' AND '$(TargetFramework)' == 'netcoreapp5.0' ">
```

Comment:
These conditions are still wrong. Looks like the baseline generator in 5.0 is not up-to-date with the one in 3.1. I'll fix this as part of this PR.